ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditTextInputConnectionWrapper.java

```diff
@@ -113,9 +113,13 @@ public boolean commitText(CharSequence text, int newCursorPosition) {
       if (key.equals("")) {
         key = BACKSPACE_KEY_VALUE;
       }
+
       dispatchKeyEventOrEnqueue(key);
     }
-
+    // Detect, if text is unicode
+    if (key.length() == 2) {
```

Comment:
WDYT about changing condition from line `112` to `if (key.length() <= 2)` instead?

Reply:
Yes, great idea. I separated it, because I wanted to make it cleaner, but it is better to remove unnecessary ifology.